diff --git a/leaderboard/utils/route_parser.py b/leaderboard/utils/route_parser.py
index c738d23..693a119 100644
--- a/leaderboard/utils/route_parser.py
+++ b/leaderboard/utils/route_parser.py
@@ -129,7 +129,7 @@ class RouteParser(object):
                 scenario_config.name = scenario.attrib.get('name')
                 scenario_config.type = scenario.attrib.get('type')
 
-                for elem in scenario.getchildren():
+                for elem in list(scenario):
                     if elem.tag == 'trigger_point':
                         scenario_config.trigger_points.append(convert_elem_to_transform(elem))
                     elif elem.tag == 'other_actor':
